<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadDatabaseWindow</class>
 <widget class="QDialog" name="LoadDatabaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Naive Bayes Classifier</string>
  </property>
  <widget class="QPushButton" name="pushButton_browseDatabase">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loadDatabase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_loadDatabase">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Database</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_databaseFilename">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
